lib/archethic/p2p/listener.ex

```diff
@@ -34,14 +38,21 @@ defmodule Archethic.P2P.Listener do
       {:ok, listener_pid} ->
         Logger.info("P2P #{transport} Endpoint running on port #{port}")
 
-        {:ok, %{listener_pid: listener_pid}}
+        {:noreply, %{listener_pid: listener_pid}}
 
       {:error, :eaddrinuse} ->
         Logger.error(
           "P2P #{transport} Endpoint cannot listen on port #{port}. Port already in use"
         )
 
         System.stop(1)
+
+      {:error, {:already_started, pid}} ->
```

Comment:
How this would happen ?

Reply:
It mainly happen during unit test

Reply:
But in the config/test.exs, there is line:

```
config :archethic, Archethic.P2P.Listener, enabled: false
```

So it should not be started by default